packages/web3-types/src/eth_types.ts

```diff
@@ -247,21 +247,51 @@ export type AccessListResult = {
 export type ValidChains = 'goerli' | 'kovan' | 'mainnet' | 'rinkeby' | 'ropsten' | 'sepolia';
 
 export type Hardfork =
-	| 'arrowGlacier'
-	| 'berlin'
-	| 'byzantium'
 	| 'chainstart'
-	| 'constantinople'
-	| 'dao'
+	| 'frontier'
 	| 'homestead'
+	| 'dao'
+	| 'tangerineWhistle'
+	| 'spuriousDragon'
+	| 'byzantium'
+	| 'constantinople'
+	| 'petersburg'
 	| 'istanbul'
+	| 'muirGlacier'
+	| 'berlin'
 	| 'london'
+	| 'altair'
+	| 'arrowGlacier'
+	| 'grayGlacier'
+	| 'bellatrix'
 	| 'merge'
-	| 'muirGlacier'
-	| 'petersburg'
-	| 'shanghai'
-	| 'spuriousDragon'
-	| 'tangerineWhistle';
+	| 'capella'
+	| 'shanghai';
```

Comment:
```
export type Hardfork = `${HardforksOrdered}`; 
```
this will auto generate type like 'chainstart' | 'frontier' ....

Reply:
Addressed [here](https://github.com/web3/web3.js/pull/6102/commits/cef42da04daf0db833f516c4bcfb7faf7b0884f9), thank you for the suggestion, much cleaner